lib/sso_openid.rb

```diff
@@ -10,20 +10,23 @@ module SsoOpenid
   class Configuration
 
     def self.openid_options
+      # Read configuration from APP_CONFIG (api-keys.yml)
+      sso_config = APP_CONFIG[:sso_openid]
+      
       {
         client_options: {
           port: 443,
           scheme: "https",
           host: SsoOpenid::Urls.sso_openid.host,
-          identifier: APP_CONFIG[:sso_openid][:client_id],
-          secret: APP_CONFIG[:sso_openid][:client_secret],
+          identifier: sso_config[:client_id],
+          secret: sso_config[:client_secret],
         },
         callback_path: SsoOpenid::Paths.callback_path,
         request_path: SsoOpenid::Paths.auth_path,
         setup_path: SsoOpenid::Paths.setup_path,
         name: :sso_openid,
         discovery: true,
-        issuer: SsoOpenid::Urls.sso_openid.discovery_endpoint,
+        issuer: SsoOpenid::Urls.sso_openid.fqdn + "/",
```

Comment:
What’s the reason for this change? It appears that fqdn and discovery_endpoint point to the same URL.